src/logger.js

```diff
@@ -0,0 +1,26 @@
+import colors from 'colors/safe';
+
+var colorList = ['blue', 'yellow', 'magenta', 'cyan'];
+
+var enabled = !!process.env.DEBUG;
+
+export default function logger (command, ...messages) {
+  if (enabled) {
+    var color = colorList[command.length % colorList.length];
+    console.log.apply(console, [colors[color](command)].concat(messages));
+  }
+}
+
+logger.print = function print (...data) {
```

Comment:
Attaching methods as a property of a function doesn't make a whole lot of sense.  It would probably make more sense to export these methods as named constants.

I.e.

```js
export const print = function (...data) {
  ...
}
```